packages/nuxt/src/module.ts

```diff
@@ -60,11 +63,24 @@ export default defineNuxtModule<VeeValidateNuxtOptions>({
       });
     }
 
-    const usingYup = checkForYup(options);
-    if (!usingYup) {
-      if (!checkForZod(options)) {
+    switch (options.typedSchemaPackage) {
+      case 'none':
+        break;
+      case 'yup':
+        checkForYup(options);
+        break;
+      case 'zod':
+        checkForZod(options);
+        break;
+      case 'valibot':
         checkForValibot(options);
-      }
+        break;
+      default:
+        if (!checkForYup(options)) {
+          if (!checkForZod(options)) {
+            checkForValibot(options);
+          }
+        }
```

Comment:
Nicely done but I don't want to be using  switches, can you change it to `if`s with early returns?

Reply:
yup, applied this fix